server/subscriber/MediaRequestSubscriber.ts

```diff
@@ -125,6 +773,28 @@ export class MediaRequestSubscriber
     }
   }
 
+  public afterInsert(event: InsertEvent<MediaRequest>): void {
+    if (!event.entity) {
+      return;
+    }
+
+    this.sendToRadarr(event.entity as MediaRequest);
+    this.sendToSonarr(event.entity as MediaRequest);
+
+    this.updateParentStatus(event.entity as MediaRequest);
+  }
```

Comment:
Do we want to make afterUpdate and afterInsert here async?

Reply:
Oh yeah! Both `afterInsert` and `afterUpdate` should be async and await the async methods inside. Otherwise, we might get unhandled promises or race conditions 🤔

I'll update it today

Reply:
@OwsleyJr Should be resolved in the latest commit